security-ldap/src/main/java/io/micronaut/configuration/security/ldap/LdapAuthenticationProvider.java

```diff
@@ -130,9 +145,7 @@ public LdapAuthenticationProvider(LdapConfiguration configuration,
 
                     LdapConfiguration.GroupConfiguration groupSettings = configuration.getGroups();
                     if (groupSettings.isEnabled()) {
-                        groups = ldapGroupProcessor.process(groupSettings.getAttribute(), result, () -> {
-                            return ldapSearchService.search(managerContext, groupSettings.getSearchSettings(new Object[]{result.getDn()}));
-                        });
+                        groups = ldapGroupProcessor.process(groupSettings.getAttribute(), result, () -> ldapSearchService.search(managerContext, groupSettings.getSearchSettings(new Object[]{result.getDn()})));
```

Comment:
Move the call to the next line for readability

Reply:
done.